app.js

```diff
@@ -4,24 +4,47 @@ var favicon = require('serve-favicon');
 var logger = require('morgan');
 var cookieParser = require('cookie-parser');
 var bodyParser = require('body-parser');
+var massive = require('massive');
 
-var routes = require('./routes/index');
+var app = module.exports = express();
 
-var app = express();
+// database setup
+var connectionString = "postgres://localhost/extreme_video_express";
+var db = massive.connectSync({connectionString: connectionString});
+app.set('db', db);
```

Comment:
I'm so jealous. This project would've helped me so much in my capstone. It took me 2 days to figure out how to set up a database and then connect with it from within my (non rails) app. So yay! This concept is really important :)